src/utils/emailHelper.js

```diff
@@ -40,7 +40,7 @@ const email = (req, url, title, msg, actionMsg) => {
                                   <tbody>
                                     <tr>
                                       <td style="box-sizing:border-box;padding:0;font-family:'Open Sans','Helvetica Neue','Helvetica',Helvetica,Arial,sans-serif;font-size:16px;vertical-align:top" valign="top">
```

Comment:
@urbanishimwe I have tested and the emails are sending but however, I thought this was supposed to be  `YML file` not `HTML file` 
<img width="347" alt="Screen Shot 2019-11-06 at 17 54 36" src="https://user-images.githubusercontent.com/41104288/68314728-569b6700-00bf-11ea-869e-30ee809dacc5.png">

Reply:
Thanks @swaibat  I'll put that into consideration